src/website/views/components/divider.pug

```diff
@@ -1,4 +1,115 @@
 ---
 title: Divider
-disabled: true
 ---
+
+p.-text Divider is a line separator with some orientation options.
+
+h3 Examples
+p.-text
+| To render a divider, use the class <code>a-divider</code>.
+
+h4 Base
+.example.-mb3
+  .-p3
+    .a-divider
+  :code(lang="html")
+    <div class="a-divider">
+    </div>
+
+h4 Vertical
+.example.-mb3
+  .-p3.-d--flex.-align-items--center
+    | option
+    .a-divider.-vertical
+    | option
+    .a-divider.-vertical
+    | option
+  :code(lang="html")
+    <div>
+      option
+    </div>
+    <div class="a-divider -vertical">
+    </div>
+    <div>
+      option
+    </div>
+    <div class="a-divider -vertical">
+    </div>
+    <div>
+      option
+    </div>
+
+h4 Dark
+.example.-mb3
+  .-p3(style="background-color:#eeeeee")
+    .a-divider.-dark
+  :code(lang="html")
+    <div class="a-divider -dark">
+    </div>
+
+h4 Base with text in a form
+.example.-mb3
+  .-p3
+    .a-divider.-textDivider or
+  :code(lang="html")
+    <div class="a-divider -textDivider">
+      or
+    </div>
+
+h4 Base within a card
+.example.-mb3
+  .-p3
+    .a-card(style="width:20rem;")
+      .a-card__content
+        p.-text.-m0
+          | Aenean pretium massa sed vehicula porta. Phasellus id metus felis.
+          | Ut felis magna, facilisis ut malesuada nec.
+        p.-text.-m0
+          | Aenean pretium massa sed vehicula porta. Phasellus id metus felis.
+          | Ut felis magna, facilisis ut malesuada nec.
+      .a-divider
+      .-d--flex.-align-items--center.-ml2.-mb1
```

Comment:
@mattnickles @davidojedacl Is it necessary to include this example here as there is a card component with footer solution that has a divider included?

Reply:
Not is necessary